<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QTreeView" name="treeView_2"/>
       </item>
       <item row="0" column="2">
        <widget class="QTreeView" name="treeView_3"/>
       </item>
       <item row="0" column="3">
        <widget class="QTreeView" name="treeView_4"/>
       </item>
       <item row="0" column="0">
        <widget class="QTreeView" name="treeView"/>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>901</width>
            <height>324</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="TableView" name="tableView"/>
           </item>
           <item>
            <widget class="TableView" name="tableView_2"/>
           </item>
           <item>
            <widget class="TableView" name="tableView_3"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <widget class="TableView" name="tableView_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>321</width>
         <height>471</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_5">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>30</y>
         <width>321</width>
         <height>471</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>widget.h</header>
   <slots>
    <signal>currentChanged(QModelIndex)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>treeView</sender>
   <signal>pressed(QModelIndex)</signal>
   <receiver>treeView_2</receiver>
   <slot>setRootIndex(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeView_2</sender>
   <signal>pressed(QModelIndex)</signal>
   <receiver>treeView_3</receiver>
   <slot>setRootIndex(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
